src/manual/products.md

```diff
@@ -72,18 +72,18 @@ In the next step, create a product of type "Selection" in the "Variants" tab, yo
 
 To each jeans article, you have to associate the combination of attributes that uniquely identifies the jeans.
 
-After saving the selection product, you will find the articles that belongs to the selection produt in the product list as well. You can also open the product details for each article directly if you click on the eye symbol in the variants tab.
+After saving the *selection* product, you will find all the articles that belong to the *selection* produt in the product list as well. You can also open the product details for each article individually if you click on the eye symbol in the variants tab.
 
-Each selection article can have own prices and images like any other product. They will be shown/used if a customer selects the variant attributes that identify the article.
+Each *selection* article can have its own texts, prices and images, like any other product. They will be shown/used if a customer selects the variant attributes that identify the article.
 
 
 ## Vouchers
 
-Voucher products are virtual products that will generate a coupon code for the bought value which is sent to the customer afterwards. The generated coupon code can be used by the customers themselves or as gifts for other persons. A voucher product can be a single article with a fixed price, an article with a [user defined price](product-details.md#price) or it can be part of a selection product (assign a variant attribute in that case).
+*Voucher* products (*not* to be confused with [*vouchers*](coupons.md)) are virtual products that will generate a *coupon code* once it has been bought. That *coupon code* will be emailed to the customer, who can use it or sent on to other persons (e.g. as a gift). A *voucher* product can be a single *article* with a fixed price, an article with a [user defined price](product-details.md#price) or part of a *selection* product (assign a variant attribute in that case).
```

Comment:
The reference to coupons.md is even more misleasing because it describes coupons and discounts, not vouchers

Reply:
It says "NOT to be confused with", because there is a "Voucher menu" and a "Voucher product". And this is indeed confusing. Is there a reason why these 2 things have the same name?